packages/components/tests/dummy/app/templates/components/tooltip.hbs

```diff
@@ -0,0 +1,244 @@
+      </R.Description>
+    </G.RadioCard>
+
+    <G.RadioCard {{on "change" this.onChange}} as |R|>
+      <Hds::TooltipButton @text="Amazon Web Services"><R.Icon @name="aws-color" /></Hds::TooltipButton>
+      <R.Label>
+        Manual peering using AWS CLI
+        <Hds::TooltipButton @text="Hello!"><FlightIcon @name="alert-circle" /></Hds::TooltipButton>
+      </R.Label>
+      <R.Description>
+        Provide you AWS CLI template to apply connection settings.
+        <Hds::TooltipButton @text="Hello!"><FlightIcon @name="alert-circle" /></Hds::TooltipButton>
+      </R.Description>
+    </G.RadioCard>
+  </Hds::Form::RadioCard::Group>
+  -->
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H3>States</Shw::Text::H3>
+
+  <Shw::Text::Body>
+    The button which serves as the opener for the tooltip has no special styles of its own other than for the focus
+    state.
+  </Shw::Text::Body>
+
+  <Shw::Text::Body>
+    <strong>Note:</strong>
+    There is a bug with the position of the tooltips below which are set to show on page load for example purposes. If
+    you scroll the page a bit they will snap back to showing as centered. (Tooltips displayed on hover or focus do not
+    have this bug.)
+  </Shw::Text::Body>
+
+  <Shw::Text::H4>With icon as opener content</Shw::Text::H4>
+
+  <Shw::Flex {{style gap="4rem"}} as |SF|>
+    {{#each @model.STATES as |state|}}
+      <SF.Item @label={{state}}>
+        {{#if (or (eq state "hover") (eq state "focus"))}}
+          <Hds::TooltipButton
+            @text="Hello!"
+            mock-state-value={{state}}
+            @extraTippyOptions={{hash showOnCreate=true}}
+          ><FlightIcon @name="alert-circle" /></Hds::TooltipButton>
+        {{else}}
+          <Hds::TooltipButton @text="Hello!" mock-state-value={{state}}><FlightIcon
+              @name="alert-circle"
+            /></Hds::TooltipButton>
+        {{/if}}
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+
+  <Shw::Text::H4>With text as opener content</Shw::Text::H4>
+
+  <Shw::Flex {{style gap="4rem"}} as |SF|>
+    {{#each @model.STATES as |state|}}
+      <SF.Item @label={{state}}>
+        {{#if (or (eq state "hover") (eq state "focus"))}}
+          <Hds::TooltipButton
+            @text="Hello!"
+            mock-state-value={{state}}
+            @extraTippyOptions={{hash showOnCreate=true}}
+          >More info</Hds::TooltipButton>
+        {{else}}
+          <Hds::TooltipButton @text="Hello!" mock-state-value={{state}}>More info</Hds::TooltipButton>
+        {{/if}}
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H3>Options</Shw::Text::H3>
+
+  <Shw::Text::H4>Placement</Shw::Text::H4>
+  <Shw::Flex as |SF|>
+    {{#each @model.PLACEMENTS as |place|}}
+      <SF.Item @label={{place}}>
+        <Hds::TooltipButton @text="Hello!" @placement={{place}}><FlightIcon @name="alert-circle" /></Hds::TooltipButton>
```

Comment:
probably would be nice to use the `showOnCreate` here too (so we can actually see the differences, and also visual regression the different variants)

(this may require a re-organization/re-layout of the elements, if you want I can help - I always enjoy working on the showcase :) )